redgenes/support_files/002.sql

```diff
@@ -14,78 +14,91 @@ create table if not exists identifier(
 
 create table if not exists md_info(
     md_id integer primary key autoincrement,
-    genome_id integer, 
-    source varchar,                 -- NCBI/external
+    entity_id integer not null, 
+    source varchar not null,                 -- NCBI/external
     source_detailed varchar,        -- zengler
     external_accession varchar,
     created_at timestamp default current_timestamp,
     modified_at timestamp default current_timestamp,
-    foreign key (genome_id) references identifier (genome_id)
+    foreign key (entity_id) references identifier (entity_id)
+);
+
+create table if not exists run_info(
+    run_id integer primary key autoincrement,
+    software varchar,
+    version varchar, 
+    parameters varchar, 
+    notes varchar,
+    created_at timestamp default current_timestamp,
+    modified_at timestamp default current_timestamp
 );
 
 -- store prodigal outputs
 create table if not exists cds_info(
     cds_id integer primary key autoincrement, 
-    genome_id integer,
-    contig_id varchar,
-    type varchar,
-    start integer,
-    end integer,
-    score real, 
-    strand varchar, 
-    phase integer, 
-    attributions integer,
+    entity_id integer not null,
+    contig_id varchar not null,
+    gene_id varchar,
+    gene_type varchar not null,
+    start integer not null,
+    end integer not null,
+    strand integer, 
+    partial varchar,
+    start_type varchar,
+    stop_type varchar,
+    rbs_motif varchar,
+    rbs_spacer varchar,
+    gc_cont real,
+    conf real,
+    score real,
+    cscore real,
+    sscore real,
+    rscore real,
+    uscore real,
+    tscore real,
+    mscore real,
```

Comment:
I'm not super familiar with the annotations but if it would be invalid for any of these to be null, I recommend placing that constraint on the column(s)

Reply:
was this something to address here or in a subsequent pr?